external-crates/move/move-compiler/tests/move_check/naming/unused_fun_tparam.move

```diff
@@ -0,0 +1,18 @@
+module 0x42::unused_fun_tparam {
```

Comment:
A small nit, but I really like splitting these tests into ones that have errors versus ones that don't. It's by no means foolproof, but it really helps me when making compiler changes, since at a glance, adding a `.exp` file is a much bigger change than modifying one.